app/controllers/gobierto_admin/base_controller.rb

```diff
@@ -64,6 +64,18 @@ def module_doc_url
       end
     end
 
+    def default_modules_home_paths
```

Comment:
We should add this to the new module documentation section.

Reply:
Submitted!